src/main/java/mate/academy/hibernate/relations/exception/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.hibernate.relations.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
+        super(message);
```

Comment:
```suggestion
    public DataProcessingException(String message, Throwable e) {
        super(message, e);
```
add exception here too, to see what went wrong in console